auction-app-frontend/src/pages/Home/Home.jsx

```diff
@@ -97,43 +99,33 @@ const Home = () => {
               <li>All Categories</li>
             </ul>
           </div>
-          <div className="highlighted-product">
-            <div className="product-container">
-              <div className="product-info body-semibold">
-                <span className="product-name">{product.name}</span>
-                <span className="product-price">
-                  Start From ${product.startPrice}
-                </span>
-                <span className="body-regular">{product.description}</span>
-                <Button label="Bid now" iconSrc={go} />
+          <Link to={`${ROUTE_PATHS.PRODUCT_DETAILS}/${product.id}`}>
```

Comment:
Click on Bid now button or product image should redirect to the product details page, not whole element.